src/telemetry/mod.rs

```diff
@@ -34,7 +34,7 @@ pub fn new(
     taps: &Arc<Mutex<tap::Taps>>,
 ) -> (Sensors, transport::Registry, tls_config_reload::Sensor, ServeMetrics) {
     let process = process::Report::new(start_time);
-    let (http_sensors, http_report) = http::new(taps);
+    let (http_sensors, http_report) = http::new(metrics_retain_idle, taps);
```

Comment:
TIOLI: Maybe the argument ought to be called `http_retain_idle` now to indicate that it only effects the HTTP metrics?